### ISSUE TITLE
Start with Gsoc

### ISSUE THREAD

**Issue by siddhant1** [scorelab/EtherBeat#14]
Since I am super interested in this project and In gsoc the goal in to design an analyzer framework over it.
How can I get started with it, since no previous repository is present now?

**Comment by ThakurKarthik** [comment]
Repository is there but just nothing is build over reactjs or graphql.Go through the codebase i have been going through the codebase for some time now.Install the tinkerpop gremlin.

**Comment by ThakurKarthik** [comment]
Yes no info from mentors too from the gsoc project page in slack channel
